### PR TITLE
net-dns/knot: Version bump

### PR DESCRIPTION
**Pull request by nemunaire**
@gentoo/proxy-maint

### CODE REVIEW

File: net-dns/knot/knot-2.3.1.ebuild [hunk 1]
```diff
@@ -35,10 +35,6 @@ DEPEND="${RDEPEND}
 	doc? ( dev-python/sphinx )
 "
 
-PATCHES=(
-	"${FILESDIR}/2.3.0-added-conversion-to-wire-format-when-writing-FNV64-hash-values.patch"
-)
-
 S="${WORKDIR}/${P/_/-}"
 
 src_configure() {
```
Comment (SoapGentoo) [inline net-dns/knot/knot-2.3.1.ebuild]: in `src_configure`, please change
`$(usex systemd --enable-systemd=yes --enable-systemd=no)`
to
`--enable-systemd=$(usex systemd)`

Comment (nemunaire) [inline net-dns/knot/knot-2.3.1.ebuild]: PR updated

**Comment by SoapGentoo** [comment]
@nemunaire could you please rebase on latest master?

**Comment by nemunaire** [comment]
Done, @SoapGentoo

**Pull request closed without merging**